pkg/testframework/config.go

```diff
@@ -39,3 +45,90 @@ func GetConfig() (*rest.Config, error) {
 
 	return nil, fmt.Errorf("could not locate a kubeconfig")
 }
+
+// Reset brand value in place before run of test
+func PutBackBrand(t *testing.T, client *Clientset, originalBrand v1.Brand) {
+	err := SetBrand(client, originalBrand)
+	if err != nil {
+		t.Fatalf("Could not reset brand back to starting value %v", err)
+	}
+}
+
+// Reset the original cm stored in the openshift-config-managed namespace, if it exist
+func PutBackManagedConfigMap(t *testing.T, client *Clientset, ns string, cm *corev1.ConfigMap) {
+	err := client.ConfigMaps(ns).Delete(consoleapi.OpenShiftConsoleConfigMapName, &metav1.DeleteOptions{})
+	if err != nil {
+		t.Fatalf("Could not delete config map %s in namespace %s , %v", cm.Name, ns, err)
+	}
+	// Check that there was a valid config map originally (use name as the validation)
+	if cm.Name != "" {
+		// Need to clear the resource version
+		cm.ResourceVersion = ""
+		_, err = client.ConfigMaps(ns).Create(cm)
+		if err != nil {
+			t.Fatalf("Could not recreate original config map %s in namespace %s , %v", cm.Name, ns, err)
+		}
+	}
+}
+
+// Set Brand on the config
+func SetBrand(client *Clientset, brand v1.Brand) error {
+	operatorConfig, err := client.Consoles().Get(consoleapi.ConfigResourceName, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	// set it
+	operatorConfig.Spec.Customization.Brand = brand
+	_, err = client.Consoles().Update(operatorConfig)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func GetBrand(t *testing.T, client *Clientset) v1.Brand {
+	cm, err := GetConsoleConfigMap(client)
+	if err != nil {
+		t.Fatalf("error: %s", err)
+	}
+
+	data := cm.Data["console-config.yaml"]
+	brandingValue := ""
+	temp := strings.Split(data, "\n")
+	for _, item := range temp {
+		if strings.Contains(item, "branding") {
+			brandingValue = strings.Split(strings.TrimSpace(item), ":")[1]
+		}
+	}
+	return stringToBrand(strings.TrimSpace(brandingValue))
+}
+
```

Comment:
If going to emulate `ApplyConfigMap()`, I'd prob return `ConfigMap, err` (keep the signature).  Returning the new created or updated configmap could be valuable.

Reply:
And I would still suggest not combining functions for `operator-config`, top level `console-config`, configmap for `console` all in the same file.   Worried about the potential mess.

Reply:
yes I agree, these functions are confusing being in one file yet being used for different configs.  I will sort them out into proper place or make new files.

Reply:
Different files.

```
// even if you
configmap.go // ApplyConfigmap() is generic for any configmap
operator-config.go
console-config.go
```
I think [this comment](https://github.com/openshift/console-operator/pull/212/files#r278976403) may prove my point.